src/config/pci-passthrough.md

```diff
@@ -0,0 +1,268 @@
+IOMMU Group 14: 
+        07:00.0 VGA compatible controller [0300]: Advanced Micro Devices, Inc. [AMD/ATI] Ellesmere [Radeon RX 470/480/570/570X/580/580X/590] [1002:67df] (rev e7) 
+        07:00.1 Audio device [0403]: Advanced Micro Devices, Inc. [AMD/ATI] Ellesmere HDMI Audio [Radeon RX 470/480 / 570/580/590] [1002:aaf0] 
+[...]
+```
+
+It is important to consider which PCI slot you use with your graphics card. Some
+PCI slots may have other peripherals in the same IOMMU group (ie. `IOMMU Group
+13` the output above), which make the slot inconvienient to pass through.
+
+### Isolating GPU
+
+Find the GPU and its associated audio device you want to pass through to your
+guest in the IOMMU group list and get the content inside the second square
+brackets (eg. `1002:67df` and `1002:aaf0`).
+
+> Note: If you have a host with two identical GPUs, you will have to follow
+> different instructions to pass through your GPU. Refer to the ArchWiki for
+> [further
+> details](https://wiki.archlinux.org/index.php/PCI_passthrough_via_OVMF#Using_identical_guest_and_host_GPUs).
+
+Add the kernel parameter (e.g. `vfio-pci.ids=1002:67df,1002:aaf0`) to your
+bootloader configuration, as described above. Do not add any other components of
+the IOMMU group, even if you later end up passing through some other device
+(e.g. a USB controller).
+
+### Checking
+
+After rebooting, use `lspci` to check if the GPU and its audio device is using
+the `vfio-pci` driver:
+
+```
+$ lspci -nnk -d 1002:67df 
+[...]
+        Kernel driver in use: vfio-pci 
+```
+
+```
+$ lspci -nnk -d 1002:aaf0 
+[...]
+        Kernel driver in use: vfio-pci 
+```
+
+## Setting up OVMF-based guest VM
+
+Install the `qemu`, `libvirt`, and `virt-manager` packages.
+
+Build and install the `ovmf` package using
+[xbps-src](https://github.com/void-linux/void-packages/#quick-start), after
+applying [these
+changes](https://github.com/void-linux/void-packages/pull/17225/files).
+
+Make sure the `libvirtd`, `virtlogd`, and `virtlockd` services are enabled.
+
+To run `virt-manager` without root, add yourself to the `libvirt` user group and
+uncomment the following lines in `/etc/libvirt/libvirtd.conf`:
+
+```
+unix_sock_group = "libvirt"
+unix_sock_ro_perms = "0770"
+unix_sock_rw_perms = "0770"
+unix_sock_admin_perms = "0770"
+unix_sock_dir = "/var/run/libvirt"
+```
+
+Log out and back in to apply the new group settings.
+
+Using `virt-manager`, create a new virtual machine, checking the "Customize
+before install" checkbox when offered. In the "Overview" section, set the
+firmware to UEFI, the chipset to Q35, and in the "CPUs" section, set "Copy host
+CPU configuration". Then complete the VM creation process.
+
+Once the VM has been created, open it, select "Add Hardware", and for "PCI Host
+Device", add the devices specified in the `vfio-pci.ids` kernel parameter. If
+you wish to add a mouse and keyboard, do so under the "USB Host Device" section.
```

Comment:
Is there any reason to complete vm creation *before* adding the GPU into the VM? Also, you should talk about removing the integration devices here, like the spice channel, virtual display, virtual GPU, virtual mouse and keyboard, and so on.